test/ptc/tools/gcs.clj

```diff
@@ -101,3 +112,41 @@
    (upload-file file bucket object (misc/get-auth-header!)))
   ([file url]
    (apply upload-file file (parse-gs-url url))))
+
+(defn gcs-cat
+  "Return the content of the GCS object at URL."
+  [url]
+  (misc/shell! "gsutil" "cat" url))
+
+(defn gcs-edn
+  "Return the JSON in GCS URL as EDN."
+  [url]
+  (-> url gcs-cat (json/read-str :key-fn keyword)))
+
+(defn wait-for-file-upload
+  "Wait for content of the GCS FILE-NAME at URL."
+  [file-name]
+  (loop [file-name file-name]
+    (let [seconds 15
+          contents (try
+                     (gcs-cat file-name)
+                     (catch Exception e
+                       (log/info (.getMessage e))
```

Comment:
```suggestion
                       (log/info e "Exception waiting for" file-name)
```
clojure.tools.logging can specifically handle throwables as the first argument to print them nicely

Reply:
I ended up removing this bit of code, but that's good to know!